ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -63,6 +63,7 @@ statement
     | INSERT INTO qualifiedName (columns)? VALUES values                    #insertValues
     | DROP STREAM (IF EXISTS)? qualifiedName (DELETE TOPIC)?                #dropStream
     | DROP TABLE (IF EXISTS)? qualifiedName  (DELETE TOPIC)?                #dropTable
+    | DROP CONNECTOR identifier                                             #dropConnector
```

Comment:
Should we also have `(IF EXISTS)` here?

Reply:
I don't feel strongly about this, but using an `IF EXISTS` will require an extra call to connect. If it doesn't exist you just get a 404 in the response, which is probably okay.